src/cultionet/data/create.py

```diff
@@ -170,31 +172,29 @@ def is_grid_processed(
     process_path: Path,
     transforms: T.List[str],
     group_id: str,
-    grid: T.Union[str, int],
-    n_ts: int
+    grid_id: T.Union[str, int],
+    uid_format: str
 ) -> bool:
     """Checks if a grid is already processed
     """
-    batch_stored = False
+    batches_stored = []
     for aug in transforms:
-        if aug.startswith('ts-'):
-            for i in range(0, n_ts):
-                train_id = f'{group_id}_{grid}_{aug}_{i:03d}'
-                train_path = process_path / f'data_{train_id}.pt'
-                if train_path.is_file():
-                    train_data = joblib.load(train_path)
-                    if train_data.train_id == train_id:
-                        batch_stored = True
+        aug_method = AugmenterMapping[aug.replace('-', '_')].value
```

Comment:
the augmenter names should be consistent, just use `_` or `-` everywhere, or better yet use enums.